### ISSUE TITLE
Accessibility Testing for MHV - SM, Medication renew via secure message flow (SM pages only)

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#123853]
### Product information

- [x] Team name, product name, and feature name have been added to the title of this issue.
- [x] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

@sarahhortonadhoc 

### Use of color and color contrast checks

- [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [x] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [x] Any charts, maps, infographics, and tables convey all information without only relying on color
- [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")
- [ ] (Mobile app only) Color contrast ratios are met for all supported dark, contrast, and inverted color modes
- [ ] (Mobile app only) All elements retain their meaning for all supported dark, contrast, and inverted color modes

### How did color testing go?

- Focus indicator does not have sufficient contrast, previously reported in [DST Staging Review: Visual focus indicator has insufficient color contrast against adjacent (background) color #4683](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4683)

### Who completed the axe scans?

@sarahhortonadhoc 

### axe checks

- [x] Each page or screen has been scanned using axe (results shared below)
- [x] axe is integrated into your end-to-end testing

### axe DevTools scan results

All pages passed.

### How did axe testing go?

All pages passed.

### Who completed the content zoom and reflow test?

@sarahhortonadhoc 

### Content zoom and reflow checks

- [x] (Web only) All page elements are readable and usable at 200% zoom
- [x] (Web only) All page elements are readable and usable at 300% zoom
- [x] (Web only) All page elements are readable and usable at 400% zoom
- [ ] (Mobile app only) All page elements are readable and usable when device text is at the maximum possible size

### How did content zoom and reflow testing go?

No issues found.

### Who completed the keyboard navigation test?

@sarahhortonadhoc 

### Keyboard navigation checks

- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [x] All elements under focus have a visible focus indicator
- [x] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

No issues found.

### Do you have any other results to share?

The following are issues identified in preparing the accessibility artifact that are not accessibility defects but that impact accessibility:

- Success banner that displays within Secure Messaging should be removed: https://github.com/department-of-veterans-affairs/va.gov-team/issues/124108
- "Provide details" alert does not persist: #124500 

The following are accessibility defects identified in preparing the accessibility artifact and their associated tickets:

- Some error notifications are not announced: #124531 
- Grouped inputs are not provided with the correct elements: #124542
- Informational icon in alert provided without alternative text: see Design System [[Bug] va-alert needs meaningful SR text to display status #5046](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/5046) and staging review ticket from Notification email update prompts [#123441](https://github.com/department-of-veterans-affairs/va.gov-team/issues/123441)
- Pages do not have descriptive titles: see staging review ticket from Oracle Health curated list [#118763](https://github.com/department-of-veterans-affairs/va.gov-team/issues/118763)

The following are accessibility defects identified in preparing the accessibility artifact that are global issues that may not have a clear path to resolution:

- For SC 1.4.12 Text Spacing (Level AA), prefilled text in inputs does not respond to user-specified settings for text spacing.
